tests/entry.rs

```diff
@@ -130,5 +180,8 @@ fn good_parent_paths_ok() {
 
     let td = t!(TempDir::new("tar"));
     t!(ar.unpack(td.path()));
-    t!(File::open(td.path().join("foo/bar")));
+    t!(td.path().join("foo").join("bar").read_link());
+    let dst = t!(td.path().join("foo").join("bar").canonicalize());
```

Comment:
Hm to be honest I'm surprised that symlinks work at all on Windows, they're supposed to be gated by default...

Want to just `#[cfg(unix)]` the symlink tests?